Chat-App/ViewControllers/OnboardingViewController.swift

```diff
@@ -9,10 +9,55 @@ import UIKit
 
 final class OnboardingViewController: UIViewController {
 
+    @IBOutlet weak var lionLabel: UILabel!
+    @IBOutlet weak var foxLabel: UILabel!
+    @IBOutlet weak var mouseLabel: UILabel!
+    @IBOutlet weak var catLabel: UILabel!
+    @IBOutlet weak var rhinocerosLabel: UILabel!
+    @IBOutlet weak var sliderView: UIView!
+    @IBOutlet weak var slider: SliderView!
+    
+    
+    
+    
     override func viewDidLoad() {
         super.viewDidLoad()
+        //TODO: create UI
+        
+        self.view.backgroundColor = .systemBlue
+        self.sliderView.layer.cornerRadius = 35
```

Comment:
We can write an `IBInspecable` properties as extensions to view, to set corner radius in storyboard

Reply:
Thank you for replay! fixed